### PR TITLE
 WIP Add .to() support for 3d channels last 

### PR DESCRIPTION
**Pull request by dmenig**
Part of a multi-PR work to fix #59168

### CODE REVIEW

File: torch/nn/modules/module.py [hunk 1]
```diff
@@ -1061,6 +1061,9 @@ def to(self, *args, **kwargs):
         .. function:: to(memory_format=torch.channels_last)
            :noindex:
 
+        .. function:: to(memory_format=torch.channels_last_3d)
```
Comment (albanD) [inline torch/nn/modules/module.py:1064]: Why add this here?
The list here is meant to show all the signatures, not all the possible arguments.

Comment (hyperfraise) [reply torch/nn/modules/module.py:1064]: I don't know what I'm doing. I thought this would be relevant. Removing it.

**Comment by albanD** [comment]
Thanks!
I don't think there is anything left in this PR, should we just close it?

**Comment by hyperfraise** [comment]
Oops ofc

**Pull request closed without merging**